.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,24 +36,45 @@ jobs:
     steps:
-        # Electron requires unprivileged user namespaces to function properly. 
-        # Disabling this security rule allows Electron to create sandboxed processes
-        # without requiring elevated privileges, which is essential for running the application.
-        # This is workaround until electron builder solves this issue in future release.
+      - name: Debug Previous Runs
+        env:
+          GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+        run: |
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          REPO=${{ github.repository }}
+          WORKFLOW_NAME="[Test] suite-desktop e2e" 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)